tests/test_physics.py

```diff
@@ -0,0 +1,76 @@
+import os.path as osp
+import random
+
+import numpy as np
+import pytest
+import quaternion
+
+import examples.settings
+import habitat_sim
+
+
+@pytest.mark.skipif(
+    not osp.exists("data/scene_datasets/habitat-test-scenes/skokloster-castle.glb")
+    or not osp.exists("data/objects/"),
+    reason="Requires the habitat-test-scenes and habitat test objects",
+)
+def test_kinematics(sim):
+    cfg_settings = examples.settings.default_sim_settings.copy()
+
+    cfg_settings[
+        "scene"
+    ] = "data/scene_datasets/habitat-test-scenes/skokloster-castle.glb"
+    # enable the physics simulator: also clears available actions to no-op
+    cfg_settings["enable_physics"] = True
+    cfg_settings["depth_sensor"] = True
+
+    # test loading the physical scene
+    hab_cfg = examples.settings.make_cfg(cfg_settings)
+    sim.reconfigure(hab_cfg)
+    assert sim.get_physics_object_library_size() > 0
+
+    # test adding an object to the world
+    object_id = sim.add_object(0)
+    assert len(sim.get_existing_object_ids()) > 0
+
+    # test kinematics
+    I = np.identity(4)
+
+    # test get and set translation
+    sim.set_translation(np.array([0, 1.0, 0]), object_id)
+    assert np.allclose(sim.get_translation(object_id), np.array([0, 1.0, 0]))
+
+    # test get and set transform
+    sim.set_transformation(I, object_id)
+    assert np.allclose(sim.get_transformation(object_id), I)
+
+    # test get and set rotation
+    Q = habitat_sim.utils.quat_from_angle_axis(np.pi, np.array([0, 1.0, 0]))
+    expected = np.eye(4)
+    expected[0:3, 0:3] = quaternion.as_rotation_matrix(Q)
+    sim.set_rotation(habitat_sim.utils.quat_to_magnum(Q), object_id)
+    assert np.allclose(sim.get_transformation(object_id), expected)
+    assert np.allclose(
+        habitat_sim.utils.quat_from_magnum(sim.get_rotation(object_id)),
+        habitat_sim.utils.quat_from_coeffs(np.array([0.0, 1.0, 0.0, 0.0])),
```

Comment:
```suggestion
        Q,
```
isn't this just `Q`?